index.js

```diff
@@ -119,4 +153,13 @@ console.log('hello world')
  * Example Output: 2, 10, 20
  */
 
-// TODO: ADD CODE HERE
+function multiplyByTwo(number) {
+    if (isNaN(number)) {
+        throw Error("An argument must be a number");
+    }
```

Comment:
This is great! It wasn't requested, but you added validation here so awesome work 🎉